hamster_lib/helpers/time.py

```diff
@@ -110,7 +111,10 @@ def extract_time_info(text):
     def get_time(time):
         """Convert a times string representation to datetime.time instance."""
         if time:
-            time = datetime.datetime.strptime(time.strip(), "%H:%M").time()
+            try:
```

Comment:
If we name the seconds group, we can decide parsing method without try-except. Alternately, we can switch based on time string length.